src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -315,6 +315,24 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket & recvData)
     }
 
 
+    // GS Reguest — another disaster addons what make client crash
```

Comment:
please reword in `// GS Reguest — another check for broken addons that cause client crash`